### ISSUE TITLE
lib.js/getdevicetypes gets wrong list of devices versus showdevicetypes

### ISSUE THREAD

**Issue by shazron** [phonegap/ios-sim#202]
showdevicetypes should rely on getdevicetypes (the logic should reside there), then print it out (the two commands do two different but related things).

**Comment by shazron** [comment]
Published 5.0.11

**Comment by aharbick** [comment]
The current version of cordova-ios uses 5.0.7.  Maybe a dumb question but how does one override that locally?

I've tried a bunch of things and phongap build ios always seems to grab a fresh copy of cordova-ios into ~/.cordova and uses ios-sim 5.0.7 which means you can't emulate iPad-Pro.

A new release of cordova-ios eventually but I'm looking for some hack / override locally to work around.

**Comment by shazron** [comment]
You can override it on a per app basis. The `node_modules` folder is located in `platforms/ios/cordova/node_modules`. 

In `platforms/ios/cordova` of your app folder, just run `npm install ios-sim` which should grab the latest. Unfortunately anything in the platforms folder is a build artifact and may change, so you may want to script this in a hook after adding a platform.

**Comment by aharbick** [comment]
OK.  That worked thanks!  However I don't think that this issue or #197 are actually fixed.

I'm still seeing "No available runtimes could be found for "iPad Pro (12.9-inch)" when trying to run the iPad-Pro device type.

```
phonegap run ios --target "iPad-Pro" --emulator
```

The reason seems to still be the inconsistency of the dash in the device/devicetype name.  If I change parseDeviceType in simctl-list-parser.js to strip dashes like so

```
...
'name': matches[1].replace(/\-/g, ' ').trim(),
...
```

Then the phonegap command works.

**Comment by shazron** [comment]
Thanks! I'll take a look,  tomorrow.

**Comment by aharbick** [comment]
FWIW...  I updated my fork to match your previous code and just included my tweak above:

https://github.com/aharbick/simctl/commit/848fba26e0275b89eeeb08ddab11e8855cc8adc3

**Comment by shazron** [comment]
This will be fixed in ios-sim 5.0.12 (I didn't modify simctl, since that lib is meant to report what is output from xcrun simctl, quirks and all).

Tested using:
```
cordova run ios --target "iPad-Pro--9-7-inch-" --emulator
cordova run ios --target "iPad-Pro" --emulator
cordova run --list
```
